### PR TITLE
Update activity-reports.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed the numbers as it is not accurate.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6590

File: microsoft-365/admin/activity-reports/activity-reports.md [hunk 1]
```diff
@@ -180,9 +180,9 @@ It'll take a few minutes for these changes to take effect on the reports in the
   
 ## What happens to usage data when a user account is closed?
 
-Whenever you close a user's account, Microsoft will delete that user's usage data within 30 days. That user will still be included in the Activity chart totals (see number 1) for the periods she was active in, but will not appear in the User Details table (see number 2).
+Whenever you close a user's account, Microsoft will delete that user's usage data within 30 days. That user will still be included in the Activity chart totals for the periods she was active in, but will not appear in the User Details table.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/activity-reports.md:183]:
```suggestion
Whenever you close a user's account, Microsoft will delete that user's usage data within 30 days. That user will still be included in the Activity chart totals for the periods the user was active in, but will not appear in the User Details table.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**